src/pages/guides/authentication/ServerToServerAuthentication/IMS.md

```diff
@@ -15,15 +15,15 @@ Parameters can be sent in the body or as query parameters. Passing parameters in
 |`grant_type`|Yes|Value should always be `client_credentials`|
 |`client_id`|Yes|The Client ID obtained from the [Adobe Developer Console](/console)|
 |`client_secret`|Yes|The value of client secret obtained from the [Adobe Developer Console](/console)|
-|`scopes`|Yes|The list of comma separated scopes you are requesting. To see the scopes available to your credential, visit your project your the [Adobe Developer Console](/console)|
+|`scope`|Yes|The list of comma separated scopes you are requesting. To see the scopes available to your credential, visit your project your the [Adobe Developer Console](/console)|
 
 
 ### Request for OAuth Server-to-Server credential
 
 ```curl
 curl -X POST 'https://ims-na1.adobelogin.com/ims/token/v3' \
   -H 'Content-Type: application/x-www-form-urlencoded' \
-  -d 'client_id={CLIENT_ID}&client_secret={CLIENT_SECRET}&grant_type=client_credentials&scope={SCOPES}'
+  -d 'client_id={client_id}&client_secret={client_secret}&grant_type=client_credentials&scope={scope}'
```

Comment:
I'd rather keep the upper-case placeholder names, I think it's somehow more "visible" that those are different that the query parameter itself.

Reply:
Makes sense. Should we make the variables upper case in all the other examples?

Reply:
IMO I'd just leave them as it is, I just thought that having `client_id={client_id}` was a bit more confusing than `client_id={CLIENT_ID}` but it's really a minor thing.